src/pages/contact/contact.html

```diff
@@ -20,7 +20,7 @@
   <ion-list>
     <ion-item *ngFor="let contact of contacts" (click)="showContactModal(contact)">
       <h2>{{contact.name}}</h2>
-      <p>{{contact.phone_number | phone}}</p>
+      <p>{{contact.phone_number}}</p>
```

Comment:
Removed pipe cause it affected the unit test, it also did not make a difference to the UI. Let me know if it is needed.